.env

```
@@ -26,7 +26,7 @@ TESTS_DB_PROJECT=tests_db_project
 TESTS_DB_LIST=tests_db_list
 TESTS_DB_EXAMPLE=tests_db_example
 
-SERVER_IA_URL="http://172.17.0.1:8082"
+SERVER_IA_URL="https://localhost:8082"
```

Change pas le .env pour tout le monde
```suggestion
SERVER_IA_URL="http://172.17.0.1:8082"
```

---

peka